modules/external-secrets-cert-manager-config.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * security/external_secrets_operator/external-secrets-log-levels.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-secrets-cert-manager-config_{context}"]
+= Configuring cert-manager for the external-secrets certificate requirements
+
+Certificate management for the `external-secrets` webhook and plugins can be assigned to `cert-manager`. This configuration is optional.
+
+When `cert-manager` is not used, `external-secrets` defaults to its own certificate management. In this mode, it automatically generates the required certificates for the webhook, while you are responsible for manually configuring certificates for the plugins.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have created the `ExternalSecretsConfig` custom resource.
+* You have installed the `cert-manager`. {cert-manager-operator} can be used. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html-single/security_and_compliance/index#cert-manager-operator-install[Installing the cert-manager Operator for Red Hat OpenShift]
+
+.Procedure
+
+. Edit the `ExternalSecretsConfig` custom resource by running the following command:
++
+[source,terminal]
+----
+$  oc edit externalsecretsconfigs.operator.openshift.io cluster
+----
+
+. Configure 'cert-manager' by editing the `spec.controllerConfig.certProvider.certManager` section as follows:
```

Comment:
Saying "Configure 'cert-manager'" reads weird to me. It should either be in backticks or not. And should there be a "the" in front?